main/zoe/api/Zoe-Intro.md

```diff
@@ -19,32 +19,41 @@ This is an introduction to Zoe, Agoric's smart contract framework. The Zoe Guide
-1. Sending out *invites* to request counter-offers.
-2. Partie *redeeming* those invites and making a counter-offer.
-3. Determining when the offer *matches* a counter-offer.
-4. Either succeeding or failing to make the transaction. 
+Once installed on Zoe, a contract is instantiated, staring a lifecycle new contract instance of:
+1. Instantiating the contract with
+   ```js
+   const initialInviteP = zoe~.makeInstance(
+     installationHandle,
+     issuerKeywordRecord,
+     terms);
+   ```
+1. As shown above, `makeInstance` returns a promise for the initial invite, shown above as `initialInviteP`.
+1. Sometimes: obtaining the `publicAPI` from that initial invite, and then obtaining invites from that.
+1. Sending out invites to request offers (including counter-offers).
+1. Parties use those invites to make offers.
+1. Determining when an offer *matches* another offer.
+1. Either succeeding or failing to make the trade.
 
 
-An offer’s completion only affects that offer and the assets involved in it; essentially its contract instance. The contract it was made on keeps running, waiting for and processing new offers. 
+An offer’s completion only affects that offer and the assets involved in it. The contract instance it was made on keeps running, waiting for and processing new offers.
 
 
-*Offer safety* is automatically enforced for parties in a contract; if an offer succeeds and completes, all parties get what they wanted. If it fails to complete successfully for any reason, parties keep what they offered.
+*Offer safety* is automatically enforced for parties in a contract; when an offer *completes*, it can complete giving the requesting party what they wanted, or refunding what it offered.
 
 
-All parties in a Zoe contract receive a *seat*, which lets them take action in that contract. They also receive a *promise* for a future payout. The contract itself does not have direct access to any parties’ assets. The payments for an offer are all run through Zoe and its *escrow* capabilities.  
+All parties in a Zoe contract instance receive an invite, which lets them make one offer to zoe for that contract instance. As a result of the offer, they receive a *promise* for a future payout. The contract itself does not have direct access to any parties’ assets. The payments for an offer are all run through Zoe, which *escrows* these assets.
 
 
-Zoe has two distinct APIs. One is client/user-facing, with `zoe.` prefixes on its commands. The other is only exposed to contracts built on Zoe (its *contract facet*), with `zcf.` prefixes on its commands. . 
+Zoe has two distinct APIs. One is client/user-facing zoe *service* object. We'll hold a remote promise for the zoe servive object in a variable named `zoeP` in our code examples.  The other is only exposed to contracts built on Zoe (its *contract facet*), which we'll hold in a variable named `zcf` in our code examples.
```

Comment:
Why is Zoe a promise and not a presence? It should be a presence, and we then don't need to call it `zoeP` - we can reference the calls like `E(zoe).makeInstance().`

Reply:
That's a good point. `E(zoe)` it is.